middlewares/validations/validations.js

```diff
@@ -0,0 +1,47 @@
+class Validations {
+  static async validateCreateUser(req, res, next) {
+    const { username, email, password } = req.body;
+    const emailRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
+    const pwdRegex = /^(?=.*?[A-Z])(?=(.*[a-z]){1,})(?=(.*[\d]){1,})(?=(.*[\W]){1,})(?!.*\s).{8,}$/;
+
+    switch (true) {
+      case email == null || email === undefined || password == null || password === undefined
+      || username == null || username === undefined:
+        return res.status(400).json({
```

Comment:
this will not execute the `break` statement in your `switch` block since you are returning early, Kindly solve this code quality issue.